cf_remote/main.py

```diff
@@ -106,6 +105,9 @@ def get_args():
                     action="store_true")
     # TODO: --region (optional)
 
+    sp = subp.add_parser("list-spawned", help="List already spawned hosts")
```

Comment:
Ah sorry, I forgot about list.

list-spawned is too long and we just talked about how this should not be related to "spawned" hosts.

Please change it to:

```suggestion
    sp = subp.add_parser("show", help="List already spawned hosts")
```

Reply:
Fixed myself.